templates/alfresco/0/docker-compose.yml

```diff
@@ -3,22 +3,36 @@ alfresco:
     CIFS_ENABLED: 'false'
     FTP_ENABLED: 'false'
   labels:
-    io.rancher.container.pull_image: always'
+    io.rancher.sidekicks: alfresco-data
   tty: true
-  image: webcenter/rancher-alfresco:v5.1.0-2
+  image: webcenter/rancher-alfresco:v5.1-201605-1
   links:
   - postgres:db
   stdin_open: true
   ports:
   - 8080:8080/tcp
+alfresco-data:
+  labels:
+    io.rancher.container.start_once: 'true'
+  image: alpine
+  volumes:
```

Comment:
Should we add a `command:` or `entrypoint:` `/bin/true`  on the data containers?